src/SearchField/index.jsx

```diff
@@ -119,6 +124,14 @@ SearchField.propTypes = {
   value: PropTypes.string,
   /**
    * Specifies the icon element(s) to use for the clear and submit buttons.
+   * The default is:
+   *
+   * ```jsx
+   * {
+   *   submit: <FontAwesomeIcon icon={faSearch} />,
+   *   clear: <FontAwesomeIcon icon={faTimes} />
```

Comment:
Paragon is moving away from font-awesome. Is there `@edx/paragon/icons` equivalent that could be used here instead?